packages/react-spring/package.json

```diff
@@ -33,5 +33,9 @@
     "@react-spring/three": "~9.5.0",
     "@react-spring/web": "~9.5.0",
     "@react-spring/zdog": "~9.5.0"
+  },
+  "peerDependencies": {
+    "react": "*",
```

Comment:
this peer dep is incorrect, I would look at the packages use of it and reference that.